From 2b520416093f5b76fc4288ec00fc9ee0d7d63e80 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Wed, 13 Jan 2016 14:48:54 +0200
Subject: [PATCH 30/30] ARM: i.MX6UL: tag kernel version -cl-som-imx6ul-1.0

Tag kernel version -cl-som-imx6ul-1.0.
This is the 1-st cl-som-imx6ul release.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/configs/cl_som_imx6ul_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/configs/cl_som_imx6ul_defconfig b/arch/arm/configs/cl_som_imx6ul_defconfig
index 8533f76..c330115 100644
--- a/arch/arm/configs/cl_som_imx6ul_defconfig
+++ b/arch/arm/configs/cl_som_imx6ul_defconfig
@@ -1,4 +1,4 @@
-CONFIG_LOCALVERSION="-cl-som-imx6ul"
+CONFIG_LOCALVERSION="-cl-som-imx6ul-1.0"
 CONFIG_KERNEL_LZO=y
 CONFIG_SYSVIPC=y
 CONFIG_FHANDLE=y
-- 
1.9.1

